src/inmanta/server/services/orchestrationservice.py

```diff
@@ -21,7 +21,7 @@
 import uuid
 from collections import abc, defaultdict
 from collections.abc import Mapping, Sequence
-from typing import Literal, Optional, cast
+from typing import Literal, Optional, Set, cast
```

Comment:
This should be imported from `collections.abc`. For most types the one from `typing` is just a deprecated alias, but `typing.Set` is even an alias to `set`, rather than `collections.abc.Set`, so it's even more important to use the correct one.

Reply:
Oh, in that case you should use built-in `set`.

Reply:
Or `collections.abc.MutableSet`, but I see little benefit to that one (in this case).